serving/docs/lmi_new/deployment_guide/configurations.md

```diff
@@ -1,6 +1,189 @@
-## Container and Model Configurations
+# Container and Model Configurations
 
-TODO
+The configuration supplied to LMI provides required and optional information that LMI will use to load and serve your model.
+LMI containers accept configurations provided in two formats. In order of priority, these are:
+
+* `serving.properties` Configuration File (per model configurations)
+* Environment Variables (global configurations)
+
+We recommend using the `serving.properties` configuration file for the following reasons:
+* Supports SageMaker Multi Model Endpoints with per model configurations 
+  * Environment Variables are applied globally to all models hosted by the model server, so they can't be used for model specific configuration
+* Separates model configuration from the SageMaker Model Object (deployment unit)
+  * Configurations can be modified and updated independently of the deployment unit/code 
+
+Environment Variables are a good option for the proof-of-concept and experimentation phase for a single model.
+You can modify the environment variables as part of your deployment script without having to re-upload configurations to S3.
+This typically leads to a faster iteration loop when modifying and experimenting with configuration values.
+
+While you can mix configurations between `serving.properties` and environment variables, we recommend you choose one and specify all configuration in that format.
+Configurations specified in the `serving.properties` files will override configurations specified in environment variables.
+
+Both configuration mechanisms offer access to the same set of configurations.
+
+If you know which backend you are going to use, you can find a set of starter configurations in the corresponding [user guide](../user_guides).
+We recommend using the quick start configurations as a starting point if you have decided on a particular backend.
+The only change required to the starter configurations is specifying `option.model_id` to point to your model artifacts.
+
+We will now cover the components of a minimal starting configuration. This minimal configuration will look like:
+
+```
+# use standard python engine, or mpi aware python engine
+engine=<Python|MPI>
+# where the model artifacts are stored
+option.model_id=<hf_hub_model_id|s3_uri>
+# which inference library to use
+option.rolling_batch=<auto|vllm|lmi-dist|deepspeed|tensorrtllm>
```

Comment:
For TensorRT-LLM rolling batch, the option should be `trtllm` instead of `tensorrtllm`

Reply:
good catch - will update here and elsewhere